src/utils.rs

```diff
@@ -188,43 +195,37 @@ mod tests {
     #[test]
     #[cfg(target_os = "linux")]
     fn should_get_linux_platform() {
-        assert_eq!(get_platform(), Ok(Platform::Linux));
+        assert_eq!(get_platform(), Platform::Linux);
     }
 
     #[test]
     #[cfg(windows)]
     fn should_get_windows_platform() {
-        assert_eq!(get_platform(), Ok(Platform::Windows));
+        assert_eq!(get_platform(), Platform::Windows);
     }
 
     #[test]
     #[cfg(target_os = "macos")]
     fn should_get_macos_platform() {
-        assert_eq!(get_platform(), Ok(Platform::MacOS));
+        assert_eq!(get_platform(), Platform::MacOS);
     }
 
     #[test]
     #[cfg(not(any(unix, windows, target_os = "macos")))]
     fn should_get_unknown_platform() {
-        assert_eq!(get_platform(), Ok(Platform::Unknown));
+        assert_eq!(get_platform(), Platform::Unknown);
     }
 
     #[test]
     fn should_get_cpu_speed() {
-        let speed = match get_cpu_speed() {
-            Ok(s) => s,
-            Err(e) => panic!("{:?}", e),
-        };
+        let speed = get_cpu_speed().unwrap();
 
         assert!(speed > 0);
     }
```

Comment:
`speed > 0` is pretty broad. I'm just curious what you ended up seeing on the M1 and whether or not that actually matched expectations for the P-core frequency.

Reply:
I'm getting 3.2GHz which matches up with what I see in `powermetrics`

```
CPU 6 frequency: 3199 MHz
CPU 7 frequency: 3212 MHz
CPU 8 frequency: 3216 MHz
CPU 9 frequency: 3218 MHz
```